tests/components-test.ts

```diff
@@ -1,9 +1,11 @@
 import { assert } from 'chai';
 import { readFileSync } from 'fs';
 import path from 'path';
-import { forEach, noop, toArray } from '../src/shared/util';
-import { getComponent, getComponentIds, getLanguageIds } from './helper/prism-loader';
+import { fileURLToPath } from 'url';
+import { forEach, noop, toArray } from '../src/shared/util.js';
+import { getComponent, getComponentIds, getLanguageIds } from './helper/prism-loader.js';
 
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
`__dirname` is not available in Node ESM files. Instead we have `import.meta.url` from which to base path handling.